src/Microsoft.AspNetCore.SignalR.Core/ClientProxyExtensions.cs

```diff
@@ -13,10 +12,11 @@ public static class ClientProxyExtensions
         /// </summary>
         /// <param name="clientProxy">The <see cref="IClientProxy"/></param>
         /// <param name="method">name of the method to invoke</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None" />.</param>
         /// <returns>A task that represents when the data has been sent to the client.</returns>
-        public static Task SendAsync(this IClientProxy clientProxy, string method)
+        public static Task SendAsync(this IClientProxy clientProxy, string method, CancellationToken cancellationToken = default)
```

Comment:
This one may be debatable. The `SendAsync` on the client can be canceled. I think these APIs should mirror as much as possible (particularly useful when Client Hubs become a thing)